test/helpers.sh

```diff
@@ -184,7 +184,7 @@ EXPECT_FILE_REGULAR () {
 #   - 1: terraform init failed
 helpers::terraform_init () {
     local -r src_dir="${1:?}"
-    terraform -chdir="${src_dir}" init -no-color -reconfigure
+    terraform -chdir="${src_dir}" init -no-color -reconfigure >/dev/null
```

Comment:
I've never had a need to see stdout from terraform init in our tests

Reply:
if it fails in PR run, how do we se what failed ? run it locally ?
I think having the output might be helpful when the test actually fails.

Reply:
it will print errors to stderr, not stdout, and it is the only command run in the test which has its output sandwiched between "test running" and "test passed/failed", so it should be pretty easy to see what failed and why. i'll take out the redirect though